pontoon/teams/views.py

```diff
@@ -270,18 +270,18 @@ def request_item(request, locale=None):
     if settings.PROJECT_MANAGERS[0] != "":
         template = get_template("teams/email_request_item.jinja")
         mail_body = template.render(payload)
+        cc = set(
+            list(locale.managers_group.user_set.values_list("email", flat=True))
+            + [user.contact_email()]
+        )
 
         EmailMessage(
             subject=mail_subject,
             body=mail_body,
             from_email=settings.LOCALE_REQUEST_FROM_EMAIL,
             to=settings.PROJECT_MANAGERS,
-            cc=locale.managers_group.user_set.exclude(pk=user.pk).values_list(
-                "email", flat=True
-            )
-            if locale
-            else "",
```

Comment:
Have you removed this check intentionally? I think we should keep it around to support the "New locale" requests.